### ISSUE TITLE
RSpec/EmptyHook auto-correct creates syntax error

### ISSUE THREAD

**Issue by Aqualon** [rubocop/rubocop-rspec#1187]
Example:
```
RSpec.describe Foo do
  describe 'foo' do
    let(:foo) { 'bar' }

    after do
    end

    it 'does some test' do
      var = true
      expect(var).to eq false
    end
  end
end
```
Running
```
rubocop spec/foo_spec.rb --only RSpec/EmptyHook -a
```
creates:
```
RSpec.describe Foo do
  describe 'foo' do
    let(:foo) { 'bar' }    it 'does some test' do
      var = true
      expect(var).to eq false
    end
  end
end
```
which is syntax error.

```
❯ rubocop -V
1.21.0 (using Parser 3.0.2.0, rubocop-ast 1.11.0, running on ruby 3.0.2 x86_64-darwin20)
  - rubocop-performance 1.11.5
  - rubocop-rails 2.12.2
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.4.0
```

**Comment by pirj** [comment]
It was fixed in #1176, but not released on RubyGems yet.
I suggest you to use the version from git.
Sorry for the trouble.

**Comment by Darhazer** [comment]
Perhaps it's time for a release :)
